### DISCUSSION TITLE
Is it possible to use a config from a supported device on a non-supported device?

### DISCUSSION THREAD

**Discussion by cryptobench** [Koenkk/zigbee2mqtt#19374] [Devices]
I have bought an LED driver which is not supported in Zigbee2MQTT. I tried following the official docs of adding support for it, but the device never sent any messages with attributes in the debug logs, so i was quiet stuck on what to do.

The service i bought the driver from then sent me this link with a similar driver from the same brand that is supported. https://www.zigbee2mqtt.io/devices/LF-AAZ012-0400-42.html

Would it be possible to get the converter from that device and try it on my new LED driver to see if it would work?

**Comment by Koenkk** [comment]
this is the converter for [LF-AAZ012-0400-42](https://gist.github.com/Koenkk/484bd5e4304b77756a5b2e88221f1f17)

**Reply by cryptobench** [reply]
Heya! I managed to find it yesterday and changed a few things and now I'm up and running. Thanks for the reply, it's much appreciated!

**Reply by Koenkk** [reply]
Please create a pull request such that device support is provided out-of-the-box.

**Reply by cryptobench** [reply]
Done. https://github.com/Koenkk/zigbee-herdsman-converters/pull/6324
